python/pyspark/sql/connect/client.py

```diff
@@ -303,6 +376,7 @@ def __init__(self, connectionString: str, userId: Optional[str] = None):
 
         self._channel = self._builder.toChannel()
         self._stub = grpc_lib.SparkConnectServiceStub(self._channel)
+        # Configure logging for the SparkConnect client.
```

Comment:
@grundprinzip @HyukjinKwon this comment is kind of dangling here... should something be called here?